manual_tests/readme.md

```diff
@@ -1,20 +1,39 @@
-# Test setup
+# Linux test setup with Vagrant
 
 This is an example setup, based on vagrant + virtualbox, that allows to easily run ansible commands to test the module.
 
-# Requirements
+## Requirements
 
 - vagrant > 2.0.0
 - virtualbox > 5.1.28
 
-# Setup
+## Setup
 
-in `$WORK_DIR/ansible-datadog/manual_tests`:
+Run the Vagrantfile defined in `ansible-datadog/manual_tests`:
 
 - provision VM: `vagrant up`
 - connect to the VM to check the configuration: `vagrant ssh`
 - destroy VM when needed: `vagrant destroy -f`
 
-in `$WORK_DIR`:
+- From `ansible-datadog`'s parent directory, run:
 
-- run ansible-playbook: `ansible-playbook ansible-datadog/manual_tests/test_5_full.yml -i ansible-datadog/manual_tests/inventory`
+```shell
+ansible-playbook ansible-datadog/manual_tests/test_5_full.yml -i ansible-datadog/manual_tests/inventory
+```
```

Comment:
Maybe we should mention that there are other files to test the other Agent versions?